### ISSUE TITLE
Visual Studio 2015 ignores all tests

### ISSUE THREAD

**Issue by PoByBolek** [nunit/nunit3-vs-adapter#699]
#### Version Info
* NUnit3TestAdapter v3.16.0
* NUnit v3.12.0
* Visual Studio Enterprise 2015 Version 14.0.25431.01 Update 3
* Repro: https://github.com/PoByBolek/NunitTestAdapterIssue
* .NET Framework 3.5 - 4.7.2

#### Problem
Since 3.16.0 Visual Studio 2015 ignores all tests with the message "Test adapter sent back a result for an unknown test case. Ignoring result for '`<test name>`'".

I think this issue occurred in commit 78300b4d6201448e2b92a3920fc060c0e5887d39 in [TestConverter.cs](https://github.com/nunit/nunit3-vs-adapter/commit/78300b4d6201448e2b92a3920fc060c0e5887d39#diff-b7a39ec0f3d5eb92159134f889fdddb3).

**Comment by mikkelbu** [comment]
This sounds like a duplicate of #692

**Comment by OsirisTerje** [comment]
Yes.  I am in the process of finishing the [PR](https://github.com/nunit/nunit3-vs-adapter/pull/698) for this.  Releasing 3.16.1 as soon as possible the coming week.  
There will be betas on the myget feed https://www.myget.org/feed/nunit/package/nuget/NUnit3TestAdapter  once the PR is merged.
